src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -9,7 +10,96 @@ void BgSpot16Bombstone_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808B5A94(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B04(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B6C(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B58(BgSpot16Bombstone* this);
+void func_808B5950(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5934(BgSpot16Bombstone* this);
+void func_808B5AF0(BgSpot16Bombstone* this);
+void func_808B5A78(BgSpot16Bombstone* this);
+
+extern Gfx D_06000C20[];
+extern Gfx D_060009E0[];
+
+typedef struct D_808B5DD8Struct {
+    s16 speedXZ;
+    s16 speedY;
+    s16 scale;
+    s16 unk_6;
+    s16 unk_8;
+    s16 unk_A;
+    s16 unk_C;
+    Vec3s rotation;
+} D_808B5DD8Struct; // size = 0x14
```

Comment:
Also these structs are probably not a thing and just s16 arrays if I had to guess. Not sure what we want to do in this case.

Reply:
If you're okay with it I think it makes sense to leave them as they are.  I went through to look at removing them and realized that it will provide more context having the named variables.

Reply:
I would go with arrays myself because when you think about it, the names don't appear in the array definitions themselves, so you have to go by index rather than name when changing data inside it or adding entries.

Reply:
And in the end it's just a way to define initial data, so it doesn't really gain much by having names in the process.

Reply:
I'm okay with it if you want to leave them as they are tho, just saying what makes the most sense to me.

Reply:
Fair enough.  I've removed them.